code/modules/paperwork/ticketmachine.dm

```diff
@@ -102,6 +104,10 @@
 	handle_maptext()
 
 /obj/machinery/ticket_machine/proc/handle_maptext()
+	if(!dispense_enabled)
+		maptext_x = 6
+		maptext = "<font face='Small Fonts' color='#0f0f0f'>OFF</font>"
```

Comment:
![image](https://user-images.githubusercontent.com/25063394/98445954-9a7ddd80-2112-11eb-8435-5ff3410dc631.png)
This needs to be brighter.

Reply:
![dreamseeker_vHQMcncN1q](https://user-images.githubusercontent.com/16618648/98446173-5aeacd80-20e1-11eb-8a87-b2165143cf2f.png)
I think I will go with this color (in red box) not totally invisible, but also doesnt draw the eye too much

Reply:
![dreamseeker_sqPg5mz7yb](https://user-images.githubusercontent.com/16618648/98446534-ba49dd00-20e3-11eb-94b0-f7e296af3b89.png)
I have made my mind adamant , Darkish RED is best option.